Github issues templates (#4386)
organicmaps/organicmaps: .github/ISSUE_TEMPLATE/bug_report.md

---
name: Bug report

> about: Create a report to help us improve OrganicMaps

Reviewer: Describe your issue in details to help us improve Organic Maps

Author: Fixed

title: ''
labels: ''
assignees: ''

---

> **Describe the bug**

Reviewer: ```suggestion
**Describe the issue**
```
Not everyone knows what a "bug" is.

> A clear and concise description of what the bug is.

Reviewer: ```suggestion
Please write here a clear and concise description of what the bug/issue is about.
```

> **Steps To Reproduce**

Reviewer: ```suggestion
**Steps to reproduce**
```
Consistency matters! ;-)

1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots or screen recordings to help explain your problem.

> **Application:**

Reviewer: ```suggestion
**System information:**
```

>  - OS: [iOS, Android, Linux, MacOS, etc]

Reviewer: ```suggestion
 - Operating system and its version: [iOS 12, Android 10, Ubuntu 22, MacOS Big Sur, etc.]
```

>  - OrganicMaps version: [you can find it clicking "?" button]

Reviewer: ```suggestion
 - Organic Maps version: [you can find it by clicking the "?" button]
```

 - Device Model: [e.g. iPhone6, Samsung S22]

**Additional context**

> Add any other context about the problem here.

Reviewer: ```suggestion
Please add any other context and important details/notes/comments about the problem here.
```
